src/error.rs

```diff
@@ -14,7 +14,7 @@ use std::{
 ///
 /// It includes cases when during validation a reference is resolved into an invalid schema,
 /// which we can't know upfront because schemas can be in remote locations.
-#[derive(Debug, PartialEq)]
+#[derive(Debug, PartialEq, Copy, Clone)]
```

Comment:
I'm not sure that `Copy` should be derived at the moment.

At the moment if we're validating with an invalid schema we're receiving a `CompilationError::SchemaError` instance which does not tell much (#57, #28).
In the long run I would expect that `CompilationError` does hold some information related to the schema that failed.

I'm mentioning this as the object might not be small and in such contexts is usually better to have the user be explicit about cloning data or passing references instead of having copies, of potentially big amount of data, done transparently.

Reply:
A fair point! Regarding the future - I was thinking about using metaschemas #28 so we can re-use `ValidationError` and it will provide more detailed information about the reason why the input schema is not valid. What do you think?

Reply:
Yeah that's what I was expecting and that was the main driver for having concerns about `Copy`.
In my mind it was reading something among the lines _ValidationError is Copy_ and relating it with the struggling of trying to remove the `ValidationError::into_owner` method 😎

Reply:
PS. Having it `Clone` is OK is more about I'm very reluctant to the idea of having `Copy` implemented except if I'm reasonable sure that the object will be very small.

Reply:
I think we can add `Clone` later when there will be progress with metaschemas